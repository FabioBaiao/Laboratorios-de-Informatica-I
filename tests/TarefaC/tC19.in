aaBaB
dbcAa
bAbCd
1 0 S
LESSLESLAESLEESSL