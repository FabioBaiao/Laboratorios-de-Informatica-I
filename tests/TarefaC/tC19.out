1 0
3 0
3 1
2 2
4 2
FIM 17
